metatensor-torch/include/metatensor/torch/atomistic/model.hpp

```diff
@@ -274,6 +276,9 @@ class METATENSOR_TORCH_EXPORT ModelMetadataHolder: public torch::CustomClassHold
     /// - "model": for reference specific to this exact model
     torch::Dict<std::string, std::vector<std::string>> references;
 
+    /// Extra metadata about this model. This can be anything.
```

Comment:
```suggestion
    /// Extra metadata about this model. This can be anything, and is intended to be used by 
    /// tools based on metatensor to store data they need.
```

Reply:
Perfect, I also updated the Python docs